feature-utils/poly-look/src/react-components/filterChips/filterChips.jsx

```diff
@@ -120,17 +123,34 @@ const ChipGroup = ({
       {extendedChipsContent.map((e) => {
         const id = chipId(e);
         return (
-          <button
-            className={isChipActive(id) ? "chip selected" : "chip"}
-            onClick={() => handleChipClick(id)}
-            key={id}
-          >
-            {e.translation || id}
-          </button>
+          <div className="chip-container" key={id}>
+            <Chip
+              id={id}
+              translation={e.translation}
+              handleClick={handleChipClick}
+              active={isChipActive(id)}
+            />
+          </div>
         );
       })}
     </div>
   );
 };
 
-export default ChipGroup;
+FilterChips.propTypes = {
+  chipsContent: propTypes.oneOfType([propTypes.string, propTypes.object])
+    .isRequired,
+  defaultActiveChips: propTypes.array,
+  onChipClick: propTypes.func,
+  exclusive: propTypes.bool,
+  theme: propTypes.string,
+  allChip: propTypes.object,
+  othersChip: propTypes.object,
+};
+
+FilterChips.defaultProps = {
+  defaultActiveChips: [],
```

Comment:
Ah, because you set them here. Good call!

Reply:
How do you think it's better to set the defaults, like this as `defaultProps` or as they were before?

Reply:
I'm not really sure, maybe @g3o314 can help you there; it's certainly a bit confusing to not find them in a single place, and best practices usually advise to initialize where you use something; I don't know if doing it separately like here will have some functional advantage. Again, I defer to @g3o314 just in case she can help with that.

Reply:
I have just made a little bit of research and `defaultProps` in React are "dying", it's not considered the best practice anymore (and apparently, they make the component render three times slower). 

Also, according to what you said, it would be less confusing for everyone if we set those defaults at the beginning.

So knowing this, I'll remove them and change it back to how it was.